includes/validation/class-amp-invalid-url-post-type.php

```diff
@@ -198,21 +200,30 @@ public static function get_invalid_url_validation_errors( $post, $args = array()
 				continue;
 			}
 			$term = get_term_by( 'slug', $stored_validation_error['term_slug'], AMP_Validation_Error_Taxonomy::TAXONOMY_SLUG );
-			if ( ! $term ) {
-				continue;
+
+			$forced_sanitization = null;
+			if ( $args['check_forced_sanitization'] ) {
+				$error = $stored_validation_error['data'];
```

Comment:
You're right but you only had this problem because the data format of the post type is changed as of 3b484e8. Now that the validation errors are stored in the post type itself, we don't need to parse it out of the term as well. So it is expected that there would be a PHP warning as you observed. But it will go away once re-checking. Since we're in alpha I didn't think it was important to ensure back-compat for a previous alpha state.